src/DynamicMvvm/Property/DynamicPropertyFactory.cs

```diff
@@ -15,16 +13,27 @@ namespace Chinook.DynamicMvvm
 	[Preserve(AllMembers = true)]
 	public class DynamicPropertyFactory : IDynamicPropertyFactory
 	{
+		private readonly bool _throwOnDisposed;
+
+		/// <summary>
+		/// Initializes a new instance of the <see cref="DynamicPropertyFactory"/> class.
+		/// </summary>
+		/// <param name="throwOnDisposed">Whether a <see cref="ObjectDisposedException"/> should be thrown when <see cref="IDynamicProperty.Value"/> is changed after being disposed.</param>
+		public DynamicPropertyFactory(bool throwOnDisposed = true)
```

Comment:
This is another well hidden breaking change. By adding a ctor, it removes the implicit parameterless ctor. Make sure you add a parameterless ctor to maintain compatibility.

Reply:
Done!